docs/doxygen/include/size_table.html

```diff
@@ -0,0 +1,25 @@
+<table>
+    <tr>
+        <td colspan="3"><center><b>Code Size of coreHTTP (example generated with GCC for ARM Cortex-M)</b></center></td>
+    </tr>
+    <tr>
+        <td><b>File</b></td>
+        <td><b><center>With -O1 Optimization</center></b></td>
+        <td><b><center>With -Os Optimization</center></b></td>
+    </tr>
+    <tr>
+        <td>core_http_client.c</td>
+        <td><center>3.1K</center></td>
+        <td><center>2.5K</center></td>
+    </tr>
+    <tr>
+        <td>http_parser.c (third-party utility)</td>
+        <td><center>15.7K</center></td>
+        <td><center>13.0K</center></td>
+    </tr>
+    <tr>
+        <td><b>Total estimates</b></td>
+        <td><b><center>18.8K</center></b></td>
+        <td><b><center>15.5K</center></b></td>
```

Comment:
I am just going to say that `<center>` is deprecated and that it might be best to use a CSS class with the `text-align: center` property instead. I hate CSS as much as you though, and I don't think browsers are going to stop rendering this tag anytime soon.